src/app/components/common/AddressKeyInput/AddressKeyInput.tsx

```diff
@@ -20,8 +20,8 @@ const AddressKeyInput = (props: ValidatorPrivateKeyInputProps) => {
     const stores = useStores();
     const classes = useStyles();
     const { address, withBeaconcha, withEtherScan, whiteBackgroundColor, withCopy } = props;
-    const etherScan = getEtherScanUrl();
-    const beaconchaBaseUrl = getBaseBeaconchaUrl();
+    const etherScan = ENV().ETHERSCAN_URL;
+    const beaconchaBaseUrl = ENV().BEACONCHA_URL;
```

Comment:
You can call `ENV()` only once and then use the value to get the rest of the fields

Reply:
done